langs.toml

```diff
@@ -104,6 +104,32 @@ foreach (String arg in args)
 size    = '4.09 MiB'
 version = 'GnuCOBOL 3.1.2.0'
 website = '//gnucobol.sourceforge.io'
+example = '''
+program-id.example.
+
+data division.
+local-storage section.
+1 n pic 99 value 0.
+1 a pic X(99).
+
+procedure division.
+*> Printing
+display "Hello World!"
```

Comment:
Looks like the last remaining failure is the lack of comma: "Hello, World!", but it's hard to see, you see can see the failure in the Checks tab.